mmv1/products/cloudidentity/api.yaml

```diff
@@ -156,10 +156,75 @@ objects:
           Google Groups are the default type of group and have a label with a key of cloudidentity.googleapis.com/groups.discussion_forum and an empty value.
 
           Existing Google Groups can have an additional label with a key of cloudidentity.googleapis.com/groups.security and an empty value added to them. This is an immutable change and the security label cannot be removed once added.
-          
+
           Dynamic groups have a label with a key of cloudidentity.googleapis.com/groups.dynamic.
-          
+
           Identity-mapped groups for Cloud Search have a label with a key of system/groups/external and an empty value.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'dynamicGroupMetadata'
+        required: false
+        input: true
+        description: |
+          Dynamic group metadata like queries and status.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'queries'
+            description: |
+              Memberships will be the union of all queries. Only one entry with USER resource is currently supported. Customers can create up to 100 dynamic groups.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Enum
+                  name: 'resourceType'
+                  description: |
+                    Resource type for the Dynamic Group Query
+                  values:
+                    - :RESOURCE_TYPE_UNSPECIFIED
+                    - :USER
+                  default_value: :RESOURCE_TYPE_UNSPECIFIED
+                - !ruby/object:Api::Type::String
+                  name: 'query'
+                  description: |
+                    Query that determines the memberships of the dynamic group.
+
+                    Examples: All users with at least one organizations.department of engineering.
+
+                    user.organizations.exists(org, org.department=='engineering')
+
+                    All users with at least one location that has area of foo and building_id of bar.
+
+                    user.locations.exists(loc, loc.area=='foo' && loc.building_id=='bar')
+
+                    All users with any variation of the name John Doe (case-insensitive queries add equalsIgnoreCase() to the value being queried).
+
+                    user.name.value.equalsIgnoreCase('jOhn DoE')
+          - !ruby/object:Api::Type::NestedObject
+            name: 'status'
+            output: true
```

Comment:
For all subfields, can you add output: true explicitly? We don't auto-propagate that, I don't think.

Reply:
Done!